Functions/BIDS/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -399,7 +400,39 @@
 	end
 end
 
-%% 11. Check if length of vector fields match the number of volumes
+%% 11. Check for Look-Locker TR
```

Comment:
Please make up your mind if this is mainly Look-Locker specific (i.e., would other vendors with Look-Locker have the same problem?) or if this is Philips specific (this is not a general Look-Locker problem, but a problem on how Philips saves the data). In the latter case, please move this to the Philips section.

Also, please explain what you're doing here in a story under the header (the independent comments are great, but don't tell the story on what goes wrong)

Reply:
Yes - will improve comments.
I have no idea how other vendors behave. And there's no Philips section. So I'll keep it here - sorry, but no point in overorganizing things. I'll keep it where it functionally belongs but of course clarify in comments that this is Philips specific.